spyderlib/widgets/helperwidgets.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sure, I think the message box with checkbox should go in here as well (the one used in the Check for updates PR)